src/cmd/merge.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/NovoNordisk-OpenSource/decentralized-tech-radar/Merger"
+	"github.com/spf13/cobra"
+)
+
+// mergeCmd represents the merge command
+var mergeCmd = &cobra.Command{
+	Use:   "merge \"<FilePath1> <FilePath2> [FilePath3, ...]\"",
```

Comment:
Don't think the arguments should be given as a string
```suggestion
	Use:   "merge <FilePath1> <FilePath2> [FilePath3, ...]",
```

Reply:
Agree with this